src/components/lab/thermometer/Scale.js

```diff
@@ -270,7 +270,7 @@ const Scale = ({
       {!showKelvinScale && CelsiusScaleComponents}
 
       {/* label notes: planets, etc */}
-      {showThermometerLabels && LabelNoteComponents}
+      {DEFAULT_SHOW_THERMOMETER_LABELS && LabelNoteComponents}
```

Comment:
Is it to force a default value? Isn't it possible to change the default value in the reducer?

Reply:
As you guessed in the PR comment `showThermometerScales` is not in any reducer. Here it was supposed to be passed down as a prop to `<Scale />` but `<Thermometer />` (which renders `<Scale />`) was passing it no such prop.